config/settings.yml

```diff
@@ -2,8 +2,12 @@
 date_format_str: '%Y-%m-%d %H:%M:%S.%L'
 rolling_indexer:
   batch_size: 500
-  pause_time_between_docs: .2
-  pause_time_between_batches: 0
+  # in seconds
+  pause_time_between_docs: 0.2
+  # commitWithin does a soft commit by default, so a little more than commitWithin is desired (seconds)
+  pause_time_between_batches: 11
```

Comment:
Isn't adding an 11 second pause going to really slow things down?

Reply:
a pause long enough so we don't repeat a 3.7 minute iteration saves us a LOT of time.  I am still experimenting with shorter pauses.   1.2 seconds was too short -- we got some repeats.